googlecloud-plugins/docs/BigQuery-batchsource.md

```diff
@@ -0,0 +1,68 @@
+# Google Big Query Batch Source
+
+
+Description
+-----------
+Batch source to use Google Big Query as a Source.
+
+
+Use Case
+--------
+This source is used whenever you need to read from a Google Bigquery table or a Bigquery result.
+
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**projectID:** The ID of the project in Google Cloud. (Macro-enabled)
+
+**jsonKeyFilePath:** the credential json key file path. (Macro-enabled)
+
+**inputTableId:** The BigQuery table to read from, in the form [optional projectId]:[datasetId].[tableId], Example:
+publicdata:samples.shakespeare. Note that if the import query is specified, this table should be a empty table
+with the query result schema. User need to first create such a table.(Macro-enabled)
+
+**tempBucketPath:** the tempory google cloud storage directory to store the intermediate result.
+Example: gs://bucketname/directoryname, the directory should not be existed. User
+should delete this directory afterward manually to avoid extra google storage charge. (Macro-enabled)
+
+**importQuery:** The SELECT query to use to import data from the specified table. Example:
+SELECT TOP(corpus, 10) as title, COUNT(*) as unique_words FROM [publicdata:samples.shakespeare].
```

Comment:
let's not use square brackets here, since it seems like an actual table name.

Also, this entire query after the * has become italicized. I don't think that's the intention. You may want to get @awholegunch to take a look at this entire file for formatting.